### DISCUSSION TITLE
Can't do simple REST API query

### DISCUSSION THREAD

**Discussion by samkhawase** [directus/directus#12165] [Archive]
Hi,
I am trying out Directus for an internal project and I keep running into roadblocks. I created a sample project using the command:

```
docker run \
-p 8055:8055 \
-e KEY=255d861b-5ea1-5996-9aa3-922530ec40b1 \
-e SECRET=6116487b-cda1-52c2-b5b5-c8022c45e263 \
-e ADMIN_EMAIL="admin@example.com" \
-e ADMIN_PASSWORD="d1r3ctu5" \
directus/directus
```
I then created 2 collections **Employer** and **Employee** with _One to Many_ relationship between them. I also granted **READ access** to **Public role** in the _settings_. I added some data to the collections as well.

However when I do a curl query to the `http://localhost:8055/items/employer` endpoint, I get a HTTP 403 FORBIDDEN error message. 

```
curl -k 'http://127.0.0.1:8055/items/employee' | jq '.'

  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   102  100   102    0     0   6684      0 --:--:-- --:--:-- --:--:-- 10200

{
  "errors": [
    {
      "message": "You don't have permission to access this.",
      "extensions": {
        "code": "FORBIDDEN"
      }
    }
  ]
}
```

I am sure I am doing something wrong here, but can't seem to figure it out. Can someone please help me here?

-- Attachments --
![Employer](https://user-images.githubusercontent.com/215221/158359998-b8d01766-e972-4027-8914-a2b866fc3f0c.png)

![Employee](https://user-images.githubusercontent.com/215221/158360045-10dce3f5-5bc5-4706-bbda-895b07a403d3.png)
![Public Role](https://user-images.githubusercontent.com/215221/158360109-6f3363c1-b0d5-4d7a-8502-4424f2e3790c.png)

**Comment by rijkvanzanten** [comment]
Did you call your collection `Employee` or `employee`? It's case sensitive, so make sure to use `/items/Employee` if you capitalized the collection name 👍🏻

**Reply by samkhawase** [reply]
I can not believe that I overlooked such a simple mistake. It works now, thanks!

**Reply by rijkvanzanten** [reply]
Sweet! Glad to here it wasn't a bug 🙊

**Reply by samkhawase** [reply]
I was 100% sure that it is a PEBKAC, now I need to find my rubber duck 🦆

**Reply by samkhawase** [reply]
BTW it would help if the error says `route does not exist` or something similar. `You don't have permission to access this` might mean that the route exists but my config is wrong somehow.

**Reply by rijkvanzanten** [reply]
> BTW it would help if the error says `route does not exist` or something similar. `You don't have permission to access this` might mean that the route exists but my config is wrong somehow.

True! This is by design though, as it means that a bad actor can't scrape and extract what collections/endpoints exist in your system. That being said, there have been discussions before around having some friendlier "dev mode", or another toggle that says "return 404s as 404s instead of 403s"

**Reply by samkhawase** [reply]
A friendlier 'dev' mode would be great for debugging.

**Comment by mdimai666** [comment]
Maybe show some link on the edit collection page?
![image](https://user-images.githubusercontent.com/16858852/173259205-c81a569d-974f-4465-b5f1-a95b7e465f0e.png)
